egg-bungeecord--geyser--floodgate.json

```diff
@@ -0,0 +1,45 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1"
+    },
+    "exported_at": "2020-11-23T02:52:35-06:00",
+    "name": "Bungeecord+Geyser+Floodgate",
+    "author": "support@pterodactyl.io",
+    "description": "For a long time, Minecraft server owners have had a dream that encompasses a free, easy, and reliable way to connect multiple Minecraft servers together. BungeeCord is the answer to said dream. Whether you are a small server wishing to string multiple game-modes together, or the owner of the ShotBow Network, BungeeCord is the ideal solution for you. With the help of BungeeCord, you will be able to unlock your community's full potential.",
```

Comment:
Could this be altered to mention geyser and floodgate get installed?

Reply:
lemme take a look, and i'll re-export them